parser.js

```diff
@@ -64,22 +67,22 @@ module.exports = class Parser {
     let token = part.tokens[0]
     let node = new Comment()
     this.init(node, part)
-    node.source.end = { column: token[5], line: token[4] }
+    node.source.end = this.getPosition(token[3])
     this.commentText(node, token)
   }
 
   /* Helpers */
 
   commentText(node, token) {
     let text = token[1]
-    if (token[6] === 'inline') {
+    if (token[4] === 'inline') {
       node.raws.inline = true
       text = text.slice(2)
     } else {
       text = text.slice(2, -2)
     }
 
-    let match = text.match(/^(\s*)([^]*\S)(\s*)\n?$/)
+    let match = text.match(/^(\s*)([\s\S]*?)(\s*)$/)
```

Comment:
Why we changed regexp here?

Reply:
There were test failures. I will test it locally with old one now once more

Reply:
> There were test failures

It is strange, because they are working right now with old RegExp in `main`.

Let’s find out the reason to be sure that we don’t have hidden bug here.

Reply:
Works with old one as well. Pushed changes